api/client/client.go

```diff
@@ -511,6 +511,10 @@ func (c *Client) dialGRPC(ctx context.Context, addr string) error {
 	dialOpts = append(dialOpts, grpc.WithContextDialer(c.grpcDialer()))
 	dialOpts = append(dialOpts,
 		grpc.WithStatsHandler(otelgrpc.NewClientHandler()),
+		grpc.WithResolvers(teleportResolverBuilder{
+			// TODO(dustin.specker): do not merch this as-is
+			PingURL: addr + "/webapi/ping",
```

Comment:
Consider using find endpoint since find is a cached result, while ping hits the auth server